app/bundles/WebhookBundle/Event/WebhookRequestEvent.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+declare(strict_types = 1);
```

Comment:
```suggestion
declare(strict_types=1);
```
Oops, sorry Zdeno, I introduced a CS issue. This fixes it.